hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -718,7 +719,12 @@ private void writeSnapshotOf(@NonNull final List<ParsedItem> postPlaceholderItem
         om.writeValue(fout, suiteSnapshots);
     }
 
-    private static Path resourceLocOf(@NonNull final String snapshotLoc, @NonNull final String suiteName) {
+    private static Path resourceLocOf(@NonNull final String snapshotLoc, @NonNull String suiteName) {
+        // If we start a test with Ethereum context we are adding a "_Eth" suffix to test name.
+        // Before we start a test we need to remove this suffix to get the correct snapshot file name.
+        if (suiteName.endsWith(HapiSuite.ETH_SUFFIX)) {
+            suiteName = suiteName.replace(HapiSuite.ETH_SUFFIX, "");
```

Comment:
Yeah, you are correct. If we have more than one `_Eth` this would fail but I think we should be safe here 😄 `String.replaceLast()`  would be really handy here 👍